cloud_AWS/terraform/module/examples/multiple-vpc-regions/main.tf

```diff
@@ -0,0 +1,68 @@
+terraform {
+  required_version = "~> 1.0"
+  required_providers {
+    aws = {
+      version = "~> 4.0"
+    }
+    kentik-cloudexport = {
+      version = "~> 0.4"
+      source  = "kentik/kentik-cloudexport"
+    }
+  }
+}
+
+module "kentik_aws_integration_vpc1" {
+  // Use the module from local filesystem
+  source = "../../"
+  // Use the module from Github
+  // source = "github.com/kentik/config-snippets-cloud/cloud_AWS/terraform/module"
+
+  providers = {
+    aws = aws.us-e1
+  }
+  rw_s3_access               = true
+  vpc_id_list                = [var.vpc_id_1]
+  s3_bucket_prefix           = "terraform-example-1"
+  s3_delete_nonempty_buckets = true
+  store_logs_more_frequently = true
+  region                     = "us-east-1"
+  create_role = false
+}
+
+module "kentik_aws_integration_vpc2" {
+  // Use the module from local filesystem
+  source = "../../"
+  // Use the module from Github
+  // source = "github.com/kentik/config-snippets-cloud/cloud_AWS/terraform/module"
+
+  providers = {
+    aws = aws.eu-c1
+  }
+
+  rw_s3_access               = true
+  vpc_id_list                = [var.vpc_id_2]
+  s3_bucket_prefix           = "terraform-example-2"
+  s3_delete_nonempty_buckets = true
+  store_logs_more_frequently = true
+  region                     = "eu-central-1"
+  create_role = false
+}
+
+module "role_and_cloud_export" {
```

Comment:
I think better name is "role_and_cloud_export**s**", because it creates as many cloud exports as items on bucket_name_list

Reply:
No longer relevant after example split into 2 submodules